knees4bees/mamminals
# Feature/data

## What does this PR do?
 - Adds data about each cat.
 - Swaps out lynx and replaces with bobcat.
 
## How should it be tested?
 - Open `index.html` in the browser and confirm that each card has different data.
